spec/spec_helper.rb

```diff
@@ -303,6 +303,14 @@ def clean_active_fedora_repository
     Valkyrie::IndexingAdapter.find(adapter_name).wipe!
   end
 
+  # Configure blacklight to use the valkyrie solr index
+  config.around(:example, index_adapter: :solr_index) do |example|
```

Comment:
Nice to have this in place.  It will help with future tests that need the valkyrie core to be primary.